doc/api/process.md

```diff
@@ -2588,6 +2588,8 @@ cases:
   and generally can only happen during development of Node.js itself.
 * `12` **Invalid Debug Argument**: The `--inspect` and/or `--inspect-brk`
   options were set, but the port number chosen was invalid or unavailable.
+* `13` **Unfinished Top-Level Await**: `await` was used outside of a function
```

Comment:
this will be used if any part of the `import()` process stalls, for example a loader returning a promise that never resolves.

Reply:
Are you suggesting to explicitly list that special case? I would prefer not to, if you’re working on an ESM loader then you  can probably imagine that `import()` stalling would look like unfinished TLA. The far more common case is going to be actual TLA usage, I assume

Reply:
No I'm just saying, with the current implementation, 13 doesn't actually imply the lack of promise resolution came from a source text module with TLA. I think the best you could do is moving the exitCode check to where module.evaluate() is called, although that still isn't inherently a source text module using TLA.

Reply:
To be clear, I consider any kind of failure to finish the main module’s evaluation, whether through TLA or not, something that should not be considered a successful operation. This is intentional, and I would be against restricting the check here to only `module.evaluate()`.

Reply:
I agree we shouldn't consider those success cases. I'm saying, the message says "Unfinished Top-Level Await" but it can be caused by other things, which is potentially confusing. It might be better to rename it to "ESM Import Stalled" or something and mention that while many things can cause it, it would probably be caused by `await (promise that never resolves)`

Reply:
I don’t think most people would know what “ESM Import Stalled” means.

Again, I think this is mostly relevant when you’re actually developing a loader or similar, and in that case you most likely don’t even need any explanation at all.

Reply:
IMO, `Stalled Top-Level Await` will be the best mix of both 😌